### DISCUSSION TITLE
When connecting digispark, After 3 seconds windows says USB device can't be recognized and Digispark blinks red

### DISCUSSION THREAD

**Discussion by NotEduCo342** [ArminJo/DigistumpArduino#61] [Q&A]
i have a Digispark rev3 board and i've been in so much trouble setting the Arduino ide up for it since the Digistump was down
after trying and succefully making it to the Upload function, when i click upload console says plug in device now (will timeout in 60 seconds) and then arduino IDE pops a notification saying Compile complete (I have not connected my digispark board yet)
after that when i connect my digispark board after 3 seconds windows says USB device can't be recognized and digispark starts blinking red

i'm using https://raw.githubusercontent.com/digistump/arduino-boards-index/refs/heads/master/package_digistump_index.json for additional boards manager
and Digistump AVR boards version 1.6.9 for the board manager and selected Digispark (default, 16.5MHz)

if i've done any steps wrong or i'm not updated with the latest things that should be done please help me

**Comment by ArminJo** [comment]
Please read the readme carefully.

**Reply by NotEduCo342** [reply]
okay i did but the problem is i don't get the working concepts of a digispark
does my code needs to be burned with a programmer everytime i want to write something into it?
i updated the Digistump AVR boards to 1.7.5 and installed the drivers also the additional boards manager and again i get an error : 
Failed uploading: uploading error: exit status 0xc0000135

**Reply by NotEduCo342** [reply]
okay i downgraded the arduino ide with the same setup and the error is gone but now again digispark is not recognized by windows but the drivers are installed and ready

**Reply by NotEduCo342** [reply]
i researched a bit further and i think it needs a new bootloader i'm gonna try to do it with ATTinyCore

**Reply by ArminJo** [reply]
> i researched a bit further and i think it needs a new bootloader i'm gonna try to do it with ATTinyCore

Thats it 👍 and good luck.
